src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -252,6 +252,8 @@ def load_arguments(self, _):
         c.argument('aci_subnet_name')
         c.argument('enable_encryption_at_host', options_list=[
                    '--enable-encryption-at-host'], action='store_true')
+        c.argument('enable_ultra_ssd', options_list=[
```

Comment:
so it cannot be updated to false anymore, right?

Reply:
`false` is by default, so it's meaningless to set as `false`, while it's still possible

Reply:
@Juliehzl could you approve since there is no blocking issue? thanks.